src/main/java/org/opensearch/ad/AnomalyDetectorProfileRunner.java

```diff
@@ -554,7 +554,7 @@ private ActionListener<RCFPollingResponse> onPollRCFUpdates(
                 .isException(
                     causeException,
                     ResourceNotFoundException.class,
-                    NotSerializedADExceptionName.RESOURCE_NOT_FOUND_EXCEPTION_NAME_UNDERSCORE.getName()
+                    NotSerializedExceptionName.RESOURCE_NOT_FOUND_EXCEPTION_NAME_UNDERSCORE.getName()
```

Comment:
This name is outside the "TimeSeries" pattern -- is that intended?

Reply:
What do you mean by "outside the "TimeSeries" pattern"? The change renamed from NotSerializedADExceptionName to NotSerializedExceptionName.